src/unlimited.cpp

```diff
@@ -1153,7 +1153,16 @@ bool IsChainNearlySyncd()
     LOCK(cs_ischainnearlysyncd);
     return fIsChainNearlySyncd;
 }
-
+uint64_t LargestBlockSeen(uint64_t nBlockSize)
+{
+     // return the largest block size that we have seen since startup
+     static uint64_t nLargestBlockSeen = BLOCKSTREAM_CORE_MAX_BLOCK_SIZE;
```

Comment:
Can you please keep your code base free of untruthful and defamatory misuse of my companies trademarks?